.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ jobs:
             - 1-dependencies-{{ checksum "package.json" }}
       - restore_cache:
           keys:
-            - 3-dependencies-example-{{ checksum "example/package.json" }}
+            - 5-dependencies-example-{{ checksum "example/package.json" }}
```

Comment:
what is this change about?

Reply:
If the contents of the node_modules change, i.e. our @appcues/react-native package source, but the version number doesn't change and thus the package.json doesn't change - I believe it can lead to unexpected issues where an old version of the package code continues getting built into the example app, rather than the new updated version. This is me be overly cautious right now to make sure we get a fresh copy in this build.

Maybe something to revisit and see if it is even worth it to cache these here, I think it is what led to the initial confusion with this release.